### ISSUE TITLE
Add support for external services in `initContainer/wait-for-service-dependencies` 

### ISSUE THREAD

**Issue by guidoiaquinti** [PostHog/charts-clickhouse#270]
### Proposed change
We've recently added an `initContainer` to wait for all the service dependencies to be ready before moving on with the pod initialization.

The current logic is pretty simple and doesn't account for external services (e.g. when using an external Redis the check gets skipped). We should add logic to account as well for external services.

### Alternative options
Do nothing.

**Comment by macobo** [comment]
Not necessarily a blocker, but a thing to consider here - can we give actionable feedback if the install fails in a simple way? 

Without `--debug` I believe it would just say rollout failed due to rollout and with `--debug` the blame would be pinned on events/web service, which doesn't seem correct either.

**Comment by guidoiaquinti** [comment]
> Not necessarily a blocker, but a thing to consider here - can we give actionable feedback if the install fails in a simple way?
> 
> Without `--debug` I believe it would just say rollout failed due to rollout and with `--debug` the blame would be pinned on events/web service, which doesn't seem correct either.

Good point. I think we can probably craft a command to pull all the logs from `initContainer` pods and "usual pods" and add it to the docs.
